hugo.yml

```diff
@@ -47,27 +47,27 @@ module:
       target:                       static
     - source:                       node_modules/@docsearch/css
       target:                       assets/scss/@docsearch/css
-    - source:                       site/static/docs/5.3/assets/img/favicons/apple-touch-icon.png
+    - source:                       site/static/docs/0.1/assets/img/favicons/apple-touch-icon.png
       target:                       static/apple-touch-icon.png
-    - source:                       site/static/docs/5.3/assets/img/favicons/favicon.ico
+    - source:                       site/static/docs/0.1/assets/img/favicons/favicon.ico
       target:                       static/favicon.ico
 
     # OUDS mod
     - source:                       node_modules/tarteaucitronjs/tarteaucitron.min.js
-      target:                       static/docs/5.3/assets/js/tarteaucitron.min.js
+      target:                       static/docs/0.1/assets/js/tarteaucitron.min.js
     - source:                       node_modules/tarteaucitronjs/lang
-      target:                       static/docs/5.3/assets/js/lang
+      target:                       static/docs/0.1/assets/js/lang
     - source:                       site/storybook
       target:                       static/storybook
 
 params:
-  subtitle:                         "Orange Boosted with Bootstrap is a Bootstrap based, Orange branded accessible and ergonomic components library…"
+  subtitle:                         "OUDS Web is a Bootstrap based, Orange branded accessible and ergonomic components library…"
   description:                      "Powerful, extensible, and feature-packed frontend toolkit. Build and customize with Sass, utilize prebuilt grid system and components, and bring projects to life with powerful JavaScript plugins."
-  authors:                          "Orange and Boosted contributors"
+  authors:                          "Orange and OUDS Web contributors"
 
-  current_version:                  "5.3.3"
-  current_ruby_version:             "5.3.3"
-  docs_version:                     "5.3"
+  current_version:                  "0.1.0"
+  current_ruby_version:             "0.1.0"
+  docs_version:                     "0.1"
```

Comment:
We will need to be very careful about this because 0.1 and 5.3 will cohabit for a bit at least. I would be much more trustful by changing all the version number at once if possible.